config/application.rb

```diff
@@ -44,5 +44,7 @@ class Application < Rails::Application
     config.paths.add Rails.root.join('lib').to_s, eager_load: true
     config.active_job.queue_adapter = :sidekiq
     config.exceptions_app = self.routes
+
+    config.active_record.yaml_column_permitted_classes = [Symbol, Date, Time, BigDecimal]
```

Comment:
If this is only needed in the testing environment, why don't you add this config on `config/environemtns/test.rb` file?

Reply:
Sorry, I wrote that was in the test because I found this issue when I was running the tests, but it still is necessary for the development and production too.
I tested now:
![image](https://user-images.githubusercontent.com/30053543/180846345-3d60282d-b827-4a38-92f3-7a86f93d8340.png)

Reply:
Alright then. No worries.